load_data.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import h5py as h5
+from pathlib import Path
+
+
+def h5_extract_data(dataset_name: str, h5_file: h5.File):
+    if "rzr" in dataset_name:
+        # 1. Extract data
+        # D
+        topics = [
+            # D
+            "input_pair/control/steer", "input_pair/control/throttle", "input_pair/control/steer",
+            # H
+            "input_pair/state/twist/linear", "input_pair/state/twist/angular",
+            # P
+            "input_pair/state/pose/position", "input_pair/state/pose/orientation", "input_pair/state/twist/linear", "input_pair/state/twist/angular",
```

Comment:
So this is my fault for not having documented / written this code very well, but: `H` is supposed to be odom information including both pose and twist. `P` is the _ground truth_ pose and twist information. Does your H5 file include ground truth pose information? For the datasets which are already in this repo, I collected that from the simulator using the `/unity_command/ground_truth/...` topics.